d2renderers/d2svg/d2svg.go

```diff
@@ -1370,7 +1370,9 @@ func drawShape(writer, appendixWriter io.Writer, diagramHash string, targetShape
 			if targetShape.FontSize != textmeasure.MarkdownFontSize {
 				styles = append(styles, fmt.Sprintf("font-size:%vpx", targetShape.FontSize))
 			}
-
+			if targetShape.Fill != "" && targetShape.Fill != "transparent" {
```

Comment:
i mean i guess if they specified transparent, we should let them?

Reply:
yes it will already be a transparent div by default when rendered, and it is just that "transparent" is the default value for text including markdown (here https://github.com/terrastruct/d2/blob/master/d2exporter/export.go#L92). This avoids unnecessary style attributes in the rendered result in all the svgs.

e.g. in https://github.com/GavinNishizawa/d2/blob/c76bc9f368be9763cca191e92b486511689d5063/e2etests/testdata/stable/markdown_stroke_fill/elk/sketch.exp.svg#L839

this adds a background color
```xhtml
<div xmlns="http://www.w3.org/1999/xhtml" class="md" style="background-color:#CEEDEE;color:red"><p>they did it in style</p>
</div>
```
vs
this already has a transparent background
```xhtml
<div xmlns="http://www.w3.org/1999/xhtml" class="md" style="color:darkorange"><h1>a header</h1>
<p>a line of text and an</p>
<pre><code>{
	indented: "block",
	of: "json",
}
</code></pre>
<p>walk into a bar.</p>
</div>
```